lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -463,6 +463,15 @@ def add_creditcard(xml, creditcard, currency=nil)
         end
       end
 
+      def add_cdpt_authentication_value(xml, creditcard)
+        xml.tag! :AuthenticationECIInd, nil
```

Comment:
Ahh yeah, sorry, the `NetworkTokenizationCreditCard` attribute is what I meant, not the option.